PhpAmqpLib/Wire/AMQPWriter.php

```diff
@@ -180,28 +180,32 @@ public function write_signed_octet($n)
     /**
      * Write an integer as an unsigned 16-bit value
      *
-     * @param $n
+     * @param int $shortValue
      * @return $this
      * @throws \PhpAmqpLib\Exception\AMQPInvalidArgumentException
      */
-    public function write_short($n)
+    public function write_short($shortValue)
```

Comment:
Something we always did at RabbitMQ was to have PRs be about one particular change. The PR mentions tests clean up but at the same time these variables are changed. I'm not sure this is the right place for these changes. Also a different PR for this would make tracing changes in the repo history easier